addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -210,51 +210,52 @@
-                <h1 class="h4 my-0">
-                    <t t-call="website_slides.slide_icon">
-                        <t t-set="icon_class" t-valuef="mr-1"/>
-                    </t>
-                    <span t-field="slide.name"/>
-                </h1>
-                <span t-if="slide.question_ids"
-                    t-att-class="'ml-2 badge %s' % ('badge-success' if channel_progress[slide.id].get('completed') else 'badge-info')">
-                    <span t-if="channel_progress[slide.id].get('completed')">
-                        <i class="fa fa-check-circle"/>
-                        <t t-esc="channel_progress[slide.id].get('quiz_karma_won', 0)"/>
-                    </span>
-                    <span t-else="" t-esc="channel_progress[slide.id].get('quiz_karma_gain', 0)"/>
-                    <span>XP</span>
-                </span>
-            </div>
-        </div>
-        <div class="col-12 col-md order-1 order-md-2 text-nowrap flex-grow-0 d-flex justify-content-end mb-3 mb-md-0">
-            <div class="btn-group flex-grow-1 flex-sm-0" role="group" aria-label="Lesson Nav">
-                <a t-att-class="'btn btn-light border %s' % ('disabled' if not previous_slide else '')"
-                    role="button" t-att-aria-disabled="'disabled' if not previous_slide else None"
-                    t-att-href="'/slides/slide/%s' % (slug(previous_slide)) if previous_slide else '#'">
-                    <i class="fa fa-chevron-left mr-2"></i> <span class="d-none d-sm-inline-block">Prev</span>
-                </a>
-                <t t-set="allow_done_btn" t-value="slide.slide_category in ['infographic', 'document', 'article', 'video'] and not slide.question_ids and not channel_progress[slide.id].get('completed') and slide.channel_id.is_member"/>
-                <a t-att-class="'btn btn-primary border text-white %s' % ('disabled' if not allow_done_btn else '')"
-                    role="button" t-att-aria-disabled="'true' if not allow_done_btn else None"
-                    t-att-href="'/slides/slide/%s/set_completed?%s' % (slide.id, 'next_slide_id=%s' % (next_slide.id) if next_slide else '') if allow_done_btn else '#'">
-                    Set Done
-                </a>
-                <a t-att-class="'btn btn-light border %s' % ('disabled' if not next_slide else '')"
-                    role="button" t-att-aria-disabled="'disabled' if not next_slide else None"
-                    t-att-href="'/slides/slide/%s' % (slug(next_slide)) if next_slide else '#'">
-                    <span class="d-none d-sm-inline-block">Next</span> <i class="fa fa-chevron-right ml-2"></i>
-                </a>
-            </div>
-            <a class="btn btn-light border ml-2" role="button" t-att-href="'/slides/slide/%s?fullscreen=1' % (slug(slide))">
-                <i class="fa fa-desktop mr-2"/>
-                <span class="d-none d-sm-inline-block">Fullscreen</span>
+    <div class="d-flex my-3 justify-content-end">
```

Comment:
Would like a fresh runbot to start checking the xml, seems like a lot of changes but maybe a lot was modified visually.
Hard to see from the base specifications so I will need to compare the screens.